azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/utils/NetUtils.java

```diff
@@ -50,16 +48,16 @@ public static String getMac() {
                 sb.append(String.format("%02X%s", mac[i], (i < mac.length - 1) ? "-" : ""));
             }
             return sb.toString();
-        } catch (UnknownHostException | SocketException e) {
-            return StringUtils.EMPTY;
+        } catch (Exception e) {
+            return "UNKNOWN_MAC";
```

Comment:
You need to change this later:
either add todo or fix it by moving GetMacHash to common-lib